### PR TITLE
Removed Social media Icon text and Added hover effect

### LINKED ISSUE #101: Removing the text with social media icons
Having only social media icons is a more subtle approach rather than adding a text with them.
Also, add some zoom animation on hover.
image

Refer to CodePen Link for more context
https://codepen.io/_rohan09_/pen/mdBqMNq

### PR DESCRIPTION
**Pull request by KhushalAbrol**
Closes #101

### CODE REVIEW

**Review by swarajpure — CHANGES_REQUESTED** [review]
Can you please add screenshots of before and after?

**Comment by KhushalAbrol** [comment]
> Can you please add screenshots of before and after?

image
Left one is before right is after

File: css/main.css [hunk 1]
```diff
@@ -168,8 +168,14 @@ nav .user-greet .user-greet-msg {
 
 .social-media-logo {
   vertical-align: middle;
-  width: 15px;
-  height: 15px;
+  width: 35px;
+  height: 35px;
+}
+
+.social-media-logo:hover{
```
Comment (ankushdharkar) [inline css/main.css:175]: Nice idea to add a subtle sizing change to reflect you can interact with the element
 
The issue is that the change looks too 'abrupt' as there's no smooth transitioning applied on them. Can we add that please?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: css/main.css (hunk 1 above)
Comment (harshith-venkatesh) [reply css/main.css:175]: @KhushalAbrol , could you address the above comment please

**Comment by rohan09-raj** [comment]
Closing this PR, as there are no further developments.

**Pull request closed without merging**